roles/quay/tasks/main.yml

```diff
@@ -68,25 +23,5 @@
   retries: 10
   delay: 30
 
-
-- name: Subscribe to container-security operator
-  k8s:
-    kubeconfig: '{{ kubeconfig }}'
-    definition:
-      apiVersion: operators.coreos.com/v1alpha1
-      kind: Subscription
-      metadata:
-        name: container-security-operator
-        namespace: openshift-operators
-      spec:
-        channel: quay-v3.3
-        installPlanApproval: Automatic
-        name: container-security-operator
-        source: redhat-operators
-        sourceNamespace: openshift-marketplace
-
```

Comment:
Do we really want to strip this out or keep it with a feature flag for when OCS is available in disconnected?